addons/kommander/1.1.0/kommander-52.yaml

```
@@ -0,0 +1,98 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-52"
+    appversion.kubeaddons.mesosphere.io/kommander: "1.1.0-rc.5"
```

```suggestion
    appversion.kubeaddons.mesosphere.io/kommander: "1.1.0-rc.6"
```
sneaking this in since i forgot to change it before 🙈